[Bug] [Alert] batchInsert execption

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

1. Configure an alert instance on the UI, select the Feishu type, and configure the alert instance.
2. Configure an alert group and select the above alert instance.
3. Configure a scheduled task, select to send notifications for both success and failure in the notification strategy, and select the above configured alert group.

Error message: When waiting for the scheduled task to execute, the following error message will be displayed on the alert server:
```
[ERROR] 2023-07-31 19:54:09.360 +0800 org.apache.dolphinscheduler.alert.service.AlertBootstrapService:[90] - Alert sender thread meet an exception
org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.binding.BindingException: Parameter 'alertSendStatuses' not found. Available parameters are [arg0, collection, list]
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:96)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:441)
	at com.sun.proxy.$Proxy133.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:272)
	at com.baomidou.mybatisplus.core.override.MybatisMapperMethod.execute(MybatisMapperMethod.java:59)
	at com.baomidou.mybatisplus.core.override.MybatisMapperProxy$PlainMethodInvoker.invoke(MybatisMapperProxy.java:148)
	at com.baomidou.mybatisplus.core.override.MybatisMapperProxy.invoke(MybatisMapperProxy.java:89)
	at com.sun.proxy.$Proxy137.batchInsert(Unknown Source)
	at org.apache.dolphinscheduler.dao.AlertDao.insertAlertSendStatus(AlertDao.java:156)
	at org.apache.dolphinscheduler.alert.service.AlertBootstrapService.send(AlertBootstrapService.java:156)
	at org.apache.dolphinscheduler.alert.service.AlertBootstrapService.run(AlertBootstrapService.java:88)
Caused by: org.apache.ibatis.binding.BindingException: Parameter 'alertSendStatuses' not found. Available parameters are [arg0, collection, list]
	at org.apache.ibatis.binding.MapperMethod$ParamMap.get(MapperMethod.java:212)
	at org.apache.ibatis.scripting.xmltags.DynamicContext$ContextAccessor.getProperty(DynamicContext.java:120)
	at org.apache.ibatis.ognl.OgnlRuntime.getProperty(OgnlRuntime.java:3341)
	at org.apache.ibatis.ognl.ASTProperty.getValueBody(ASTProperty.java:121)
	at org.apache.ibatis.ognl.SimpleNode.evaluateGetValueBody(SimpleNode.java:212)
	at org.apache.ibatis.ognl.SimpleNode.getValue(SimpleNode.java:258)
	at org.apache.ibatis.ognl.Ognl.getValue(Ognl.java:586)
	at org.apache.ibatis.ognl.Ognl.getValue(Ognl.java:550)
```

### What you expected to happen

no error

### How to reproduce

as mentioned above


### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.java
+++ b/dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.java
@@ -27,7 +27,7 @@ import com.baomidou.mybatisplus.core.mapper.BaseMapper;
 
 public interface AlertSendStatusMapper extends BaseMapper<AlertSendStatus> {
 
-    int batchInsert(List<AlertSendStatus> alertSendStatuses);
+    int batchInsert(@Param("alertSendStatuses") List<AlertSendStatus> alertSendStatuses);
 
     void deleteByAlertIds(@Param("alertIds") List<Integer> alertIds);
 }

--- a/dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.xml
+++ b/dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.xml
@@ -27,7 +27,7 @@
             #{alertSendStatus.alertPluginInstanceId},
             #{alertSendStatus.sendStatus},
             #{alertSendStatus.log},
-            #{alertSendStatus.createTime},
+            #{alertSendStatus.createTime}
         </foreach>
 
     </insert>

